### ISSUE TITLE
IJPOpen on the new PEP-Web

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-Configuration#14]
While we have a general plan for adding IJPOpen to the new PEP-Web, I need to make sure we are all in sync, and there's no better time than now, since I have three volumes of IJPOpen articles already keyed and ready to be added to the live database (and more coming in from Aptara).  I feel GitHub is the best place to nail down and track these details, and it can put them in one place, whereas the emails we've exchanged do not.  This issue is meant to cover and integrate those, with direction to the responsible parties referenced in the specific issue.

Note that @rickardnyman is in charge of coordinating this overall per the specs.  Right now, my interest is focused on how I add these new issues to the database.  But here's a quick summary, and the rest can be handled and assigned in responses.

Tasks:

1) @rickardnyman to coordinate the development of the paper submission and management system, as per the old system.  He has the PHP file from that system and of course the old system being still live as guides

2) @rickardnyman to coordinate the integration of Disqus into the PEP-Web Client

3) @rickardnyman to work out whether or not IJPOpen still requires separate user registration.  If so, that will need to be coordinated, if not developed by @jwoosnam.  For now, I'm putting IJPOpen in the PEPFuture category, but it probably belongs in PEPArchive or PEPSpecial and just needs to be rights managed by PaDS wherever it is.  

4) @nrshapiro to integrate IJPOpen as a journal into the API Server load and retrieval process

5) @jwoosnam to make any necessary changes to PaDS to control access per above.

Please submit issues, ideas, and refinements in responses below.

cc: @davidtuckett @nadinelevinson

**Comment by jwoosnam** [comment]
My response to this is no different to #13 .  When will this content be available to Stage so we can test there?

**Comment by nrshapiro** [comment]
@jwoosnam 

Yes, I did run this through the PEPFuture process for now since we are not ready to release IJPOpen as it will be...a special class of article.

It's up on Stage now.  I updated the server supplied messages for it.  The others are not correct for IJPOpen.

But as a PEPFuture article, the permissions seem to be working, and only abstracts are visible.  

![image](https://user-images.githubusercontent.com/4837554/141328200-d3e3c9bb-af3e-4b48-9281-a3d3997b54db.png)
